common/Makefile.am

```diff
@@ -18,6 +18,12 @@ AM_CPPFLAGS = \
   -DXRDP_PID_PATH=\"${localstatedir}/run\" \
   -DXRDP_LOG_PATH=\"${localstatedir}/log\"
 
+if XRDP_DEBUG
+AM_CPPFLAGS += -DXRDP_DEBUG
+else
+AM_CPPFLAGS += -DXRDP_NODEBUG
```

Comment:
Either you misread my comment, or you quoted wrong piece of code. How would this code break if XRDP_NODEBUG is not defined? I only see XRDP_NODEBUG in Makefile.am, not in the actual code.